CMake/cdat_modules_extra/git_clone.sh.in

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 cd @CMAKE_INSTALL_PREFIX@
-@GIT_EXECUTABLE@ clone --depth 1 -b @BRANCH@ @GIT_URL@ @GIT_TARGET@
+"@GIT_EXECUTABLE@" clone --depth 1 -b @BRANCH@ @GIT_URL@ @GIT_TARGET@
```

Comment:
also shouldn't it be: "@GIT_TARGET@" just for safety in case there's a space or is it not possible?